site/content/en/latest/user/quickstart.md

```diff
@@ -83,7 +92,9 @@ Curl the example app through Envoy proxy:
 curl --verbose --header "Host: www.example.com" http://$GATEWAY_HOST/get
 ```
 
-## Clean-Up
+{{% /tab %}}
```

Comment:
imo this doesnt need to be a tab

Reply:
+1, I'll correct it in the next commit

Reply:
@arkodg don't need the tab function, do I need to delete the html template file corresponding to the tab? It might be useful in #3133

Reply:
not sure I understand, I was just referring to keeping `Clean-Up` the way it is and only using tabs for the Traffic Testing section

Reply:
got it.